### ISSUE TITLE
Suggestion with regards to the OS projects being decaffeinated nightly.

### ISSUE THREAD

**Issue by madskonradsen** [decaffeinate/decaffeinate#1355]
Suggestion with regards to the OS projects being decaffeinated nightly:
Settle on a fixed commit rather than pulling the newest stuff every day. It just seems like the patches fail way to often, and causes a ton of maintenance to be done.

**Comment by eventualbuddha** [comment]
For some of the projects it actually does pin specific commits, like with hubot and atom. The others probably do just pull in master, though @alangpierce would know better.

This does bring up a good question, though. What do we want to do long-term with the table in the README and the decaffeinate-examples stuff? I don't think @alangpierce or I are interested in maintaining it anymore as we mostly consider decaffeinate "done".

**Comment by madskonradsen** [comment]
Old tests are better than broken tests/patches :)

**Comment by alangpierce** [comment]
When I first set it up, it felt right to pull in master to keep up with project changes (like maybe a CoffeeScript upgrade). Definitely agreed that it would be nice to just pin all versions and get a working build that's hopefully stable. In Sucrase (another project of mine), I set up a similar system that runs in each CI build, and I just pinned all versions from the start, which seems like it was a good idea ( https://github.com/alangpierce/sucrase/tree/master/example-runner/example-configs ). With the decaffeinate build, I think I mostly got tired of maintaining the build, though it has been on my to-do list for a while to get it working again. I think most of them can be fixed and pinned and not need to be maintained anymore (and maybe it can run weekly instead of daily or something).

FWIW, Atom doesn't work with the version pinned because the build step downloads specific packages from apm, and apm support downloading very old packages. So even version pinning won't work with complex builds.
